<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1073</width>
    <height>810</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>940</x>
      <y>30</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>950</x>
      <y>710</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>pre-alpha version</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>40</y>
      <width>271</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>CAN BUS Manager</string>
    </property>
    <property name="textFormat">
     <enum>Qt::PlainText</enum>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>960</x>
      <y>120</y>
      <width>95</width>
      <height>111</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QPushButton" name="StartButton">
       <property name="text">
        <string>Start</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="SettingsButton">
       <property name="text">
        <string>Settings</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="ExitButton">
       <property name="text">
        <string>Exit</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>960</x>
      <y>250</y>
      <width>95</width>
      <height>121</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_11">
     <item>
      <widget class="QPushButton" name="pushButton_5">
       <property name="text">
        <string>L Graph</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_3">
       <property name="text">
        <string>Status</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_2">
       <property name="text">
        <string>Reset</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QTextEdit" name="TimesDisplay">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>440</y>
      <width>331</width>
      <height>201</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit11">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>480</y>
      <width>113</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit12">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>480</y>
      <width>113</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit13">
    <property name="geometry">
     <rect>
      <x>740</x>
      <y>480</y>
      <width>113</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit21">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>510</y>
      <width>113</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit22">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>510</y>
      <width>113</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit23">
    <property name="geometry">
     <rect>
      <x>740</x>
      <y>510</y>
      <width>113</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit31">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>540</y>
      <width>113</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit32">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>540</y>
      <width>113</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit33">
    <property name="geometry">
     <rect>
      <x>740</x>
      <y>540</y>
      <width>113</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>464</x>
      <y>480</y>
      <width>21</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>S1</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>464</x>
      <y>510</y>
      <width>21</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>S2</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_11">
    <property name="geometry">
     <rect>
      <x>464</x>
      <y>540</y>
      <width>21</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>S3</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_12">
    <property name="geometry">
     <rect>
      <x>554</x>
      <y>450</y>
      <width>21</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>T1</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_13">
    <property name="geometry">
     <rect>
      <x>675</x>
      <y>450</y>
      <width>20</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>T2</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_14">
    <property name="geometry">
     <rect>
      <x>794</x>
      <y>450</y>
      <width>21</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>T3</string>
    </property>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>278</x>
      <y>100</y>
      <width>671</width>
      <height>311</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>ID</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>Lenght</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_6">
         <property name="text">
          <string>Data</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_13">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_5">
           <item>
            <widget class="QPushButton" name="TestModOnButton">
             <property name="text">
              <string>Test mod ON</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="TestModOffButton">
             <property name="text">
              <string>Test mod OFF</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="GetTimeAverageButton">
             <property name="text">
              <string>Get T aver</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_7">
           <item>
            <widget class="QPushButton" name="TransmitButton">
             <property name="text">
              <string>Transmit</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="ReadButton">
             <property name="text">
              <string>Read</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="StopButton">
             <property name="text">
              <string>Stop</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_12">
         <item>
          <widget class="QCheckBox" name="ListenModBox">
           <property name="text">
            <string>Listen mod</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="checkBox_2">
           <property name="text">
            <string>Show</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QTextEdit" name="textEdit">
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>100</y>
      <width>261</width>
      <height>314</height>
     </rect>
    </property>
    <property name="title">
     <string>Options</string>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_4">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_5">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_6">
           <item>
            <widget class="QLabel" name="label_8">
             <property name="text">
              <string>Choose channel</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QListWidget" name="listWidget">
             <item>
              <property name="text">
               <string>Channel 0</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Channel 1</string>
              </property>
             </item>
            </widget>
           </item>
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_7">
             <item>
              <widget class="QLabel" name="label_7">
               <property name="text">
                <string>Choose boudrate</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QListWidget" name="listWidget_2">
               <item>
                <property name="text">
                 <string>125 kbit</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>500 kbit</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>1000 kbit</string>
                </property>
               </item>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_8">
           <item>
            <widget class="QRadioButton" name="radioButton">
             <property name="text">
              <string>11 bit</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QRadioButton" name="radioButton_2">
             <property name="text">
              <string>29 bit</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QRadioButton" name="radioButton_3">
             <property name="text">
              <string>11/29 bit</string>
             </property>
            </widget>
           </item>
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_9">
             <item>
              <layout class="QVBoxLayout" name="verticalLayout_10"/>
             </item>
            </layout>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QPushButton" name="Connect">
         <property name="text">
          <string>Connect</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QLineEdit" name="lineEdit_L3">
    <property name="geometry">
     <rect>
      <x>896</x>
      <y>540</y>
      <width>113</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_L1">
    <property name="geometry">
     <rect>
      <x>896</x>
      <y>480</y>
      <width>113</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_L2">
    <property name="geometry">
     <rect>
      <x>896</x>
      <y>510</y>
      <width>113</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_15">
    <property name="geometry">
     <rect>
      <x>950</x>
      <y>450</y>
      <width>21</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>L</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1073</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuOptions">
    <property name="title">
     <string>Options</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuOptions"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>ExitButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>857</x>
     <y>276</y>
    </hint>
    <hint type="destinationlabel">
     <x>841</x>
     <y>409</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
